analysis/monkmistweaver/src/modules/shadowlands/covenant/FallenOrderAverageHPOfTargetOnCast.tsx

```diff
@@ -65,9 +65,12 @@ class FallenOrderAverageHPOfTargetOnCast extends Analyzer {
   }
 
   statistic() {
-    const averageOfAll = this.getAverage(this.hpOfTargetOnCast);
```

Comment:
I'd much rather have an if statement that disables the stat if the length is zero than 20 billion ternaries